lib/menu.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'dish'
+
+class Menu
+  def dishes
+    @dishes ||= []
+  end
+
+  def dish_prices
+    @dish_prices ||= []
+  end
+
+  def add dish
+    dishes << dish.name
+    dish_prices << dish.price
+  end
+
```

Comment:
If add() takes a dish object as an argument, that object disappears because we are storing just the names and the prices into some lists.